src/_tutorials/language/futures.md

```diff
@@ -11,22 +11,41 @@ nextpage:
 <div class="mini-toc" markdown="1">
   <h4>What's the point?</h4>
 
-  * Dart is single-threaded.
-  * Synchronous code can make your program freeze.
-  * Use `Future` objects (_futures_) to perform asynchronous operations.
-  * Use `await` in an async function to suspend execution until a future completes.
-  * Or use the `then()` method.
-  * Use try-catch expressions in async functions to catch errors.
-  * Or use the `catchError()` method.
-  * You can chain futures to run asynchronous functions in order.
+  * Dart code runs straight through, in a single "thread" of execution.
+  * Code that blocks the thread of execution can make your program freeze.
+  * `Future` objects (_futures_) represent asynchronous operations —
```

Comment:
... represents the results of asynchronous operations

(Pet peeve, sorry, but I do think it's an important distinction).